backend/src/database/databaseconnection.ts

```diff
@@ -0,0 +1,77 @@
+import MySQL from "mysql";
+import * as fs from "fs";
+
+
+export type DBconfig = {
+    host: string,
+    port: number,
+    username: string,
+    password: string,
+    database: string,
+    supportBigNumbers?: boolean,
+    connectionLimit?: number
+}
+
+type QueryResponse = {
+    error: boolean,
+    error_message: string,
+    rows: any[]
+}
+
+type UpdateResponse = {
+    error: boolean,
+    error_message: string,
+    fieldCount: number,
+    affectedRows: number,
+    insertId: number,
+    warningCount: number,
+    message: string,
+    changedRows: number,
+    rows?:any
+}
+
+//export const config: DBconfig = JSON.parse(fs.readFileSync("./database/database.json","utf-8"));
+import * as config from "./database.json";
+
+
+const pool : MySQL.Pool = MySQL.createPool({
+    host: config.host,
+    port: config.port as unknown as number,
```

Comment:
Wenn du `as uknown as number` schreiben musst, weißt du schon das du was falsch machst.

Besser:
Typ in JSON Datei ändern zu int, also `"` weg.